project-communications/newsletter/newsletter-012921.md

```diff
@@ -0,0 +1,181 @@
+Alice founded it almost 20 years ago. 
+The organisation is  called 'Pathways to Independence'. 
+It provides support and accommodation to young refugees and asylum seekers, unaccompanies 16+. 
+You can find our more on the organisation's [website](https://www.pathwaystoindependence.org.uk/).
+
+Here she is; 
+
+![](https://i.imgur.com/jJUd1uJ.jpg)
+
+And here she is in her own words: 
+
+>"I'm a 47 year-old late-diagnosed autistic woman and mother. 
+>I am actually multiply neurodivergent: I'm autistic, and I have attention differences. 
+>My proudest >achievement is that I'm a Mum to my daughter Isabell, who is also multiply neurodivergent. 
+>We live together in our home in Brighton with our cat Belle, who I say to my daughter is also autistic - and that's why I picked her, because she's very >particular." 
+
+>"I think something interesting about my profile, which I think is becoming more and more common now, is I've lived life both sides of the river. 
+>I've lived for 46 >years as a neurotypical person, and I've only lived the past year and a half knowing that I was autistic. 
+>Getting a diagnosis was life-changing for me. 
+>My motivation wasn't because I needed extra support, it was more because of curiosity, and because of my daughter. 
+>My diagnosis came as a positive thing, as a quest of curiosoty and self-knowledge, rather than being because I was experiencing problems in my life. So that's actually quite an unusual thing. 
+>I wasn't "diagnosed" in that pathology paradigm of something being wrong. 
+>So for me, I think that's quite an interesting angle, that my diagnosis was about something more, it was about my strengths - but also so that I could locate some of the challenges that I'd previously experienced in my neurotypical life.""
+
+>"For me it's just a different way of being in the world, which describes my relationship to the world and people. 
+>I always grew up feeling different, and not really understanding why. 
+>I had different feelings about things, different reactions to things, and I felt like a refugee actually - which is quite interesting given my work. 
+>I felt dislocated and alienated. 
+>So when I got my diagnosis I could make sense of why I felt and thought differently. 
+>It was validating. 
+>A lot of the things I would criticise myself for, like that I couldn't understand things in the same way other people did, or that I couldn't think about things >in the same way - can lead to really negative feelings about yourself, which can become a self-perpetuating cycle." 
+
+>"Some autistic people have quite switched off experiences, but for me, I experience life in glorious technicolour. 
+>Don't get me wrong, I'm not seeing things dancing - it's rooted in reality, but I see and feel things very intensely. 
+>My particular profile is I'm hyper-sensitive and hyper-emotional.
+>So an example of that would be that if I saw a suffering animal or a dead animal, I would experience pain. 
```

Comment:
perhaps a trigger warning (it's not overly graphic but it might help). 

An option could be to have a sentence for the introduction of 
_Hear about Alice, her journey of getting diagnosed, and how she experiences what she calls her "technicolour" life:_

Reply:
the TW could be "emotional imagery" and could come above this section of text.